benchmarks/CMakeLists.txt

```diff
@@ -2,6 +2,17 @@
 
 project(RMM_BENCHS LANGUAGES C CXX CUDA)
 
+if(CUDA_STATIC_RUNTIME)
+    message(STATUS "Enabling static linking of cudart")
+    set(CUDART_LIBRARY CUDA::cudart_static)
+else()
+    set(CUDART_LIBRARY CUDA::cudart)
+endif(CUDA_STATIC_RUNTIME)
```

Comment:
As said above, this was in the right place before. You can kinda tell that you removed it once, but now have to add it in every place that uses rmm. cmake allows you to do it once.

Reply:
Doesn't seem to work without it now...

Reply:
removed